tests/e2e/import.test.js

```diff
@@ -0,0 +1,199 @@
+describe('Import/Create', () => {
+  let chromeBrowser;
+  let page;
+
+  const passwordErrorLabel = '#options-root > div > div.MuiContainer-root.MuiContainer-maxWidthSm > div > div.MuiGrid-root.makeStyles-passwordError-4.MuiGrid-item.MuiGrid-grid-xs-12 > p';
+
+  const badSeedphrase = 'sadf adsf adfdfasd adfad adfafd sdfsd sdfsdf sdfds sdfd sdf sfsfs sdfadsf';
+
+  const getButtonCollection = async (page) => {
+    return await page.$$('button');
+  }
+
+  const getImportButton = async (page) => {
+    const buttonCollection = await getButtonCollection(page);
+    return buttonCollection[0];
+  };
+
+  const getCreateButton = async (page) => {
+    const buttonCollection = await getButtonCollection(page);
+    return buttonCollection[1];
+  };
+
+  beforeAll(async () => {
+    chromeBrowser = await setupChrome();
+  });
+  
+  beforeEach(async () => {
+    page = await chromeBrowser.newPage();
+    await page.goto(chromeData.optionsUrl);
+  });
+
+  afterEach(async () => {
+    await page.close();
+  });
+
+  afterAll(async () => {
+    await chromeBrowser.close();
+  });
+
+  describe('Import', () => {
+    beforeEach(async () => {
+      const importButton = await utils.getButtonWithIndex(page, 0);
+      await importButton.click();
+    });
+
+    test('Fails on incorrect seedphrase', async () => {
+      const importInputElement = await utils.getElement(page, 'textarea');
+      await importInputElement.type(badSeedphrase);
+
+      const submitImport = await utils.getButtonWithIndex(page, 0);
+      await submitImport.click();
+
+      const updatedImportBtn = await utils.getButtonWithIndex(page, 0);
+      const importButtonClassName = await (await updatedImportBtn.getProperty('className')).jsonValue();
+      const importButtonClassNameArray = importButtonClassName.split(' ');
+
+      expect(importButtonClassNameArray).toEqual(expect.arrayContaining(['Mui-disabled']));
```

Comment:
Should we also check for the error message here ?

Reply:
I think we're currently not showing any error message here 
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/44899916/169573639-ba6631ef-d835-4d1a-b7ce-bd4db6503f08.png">

Reply:
we should I think @riveign